src/pages/TestOrder.jsx

```
@@ -48,288 +54,351 @@ export default function TestOrder() {
     }
   };
 
-  // 주문 상태 변경
-  const handleStatusChange = async (orderId, status) => {
-    if (!orderId) {
-      alert('주문을 먼저 선택해주세요.');
-      return;
+  // 장바구니 → 결제 → 성공 페이지 전체 플로우 테스트
+  const testCompletePaymentFlow = async () => {
+    setIsPaymentTesting(true);
+    addTestResult('전체 결제 플로우', 'warning', '테스트 시작...');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)